src/test/rules/helper.ts

```diff
@@ -24,13 +25,16 @@ export function testScript(rule: string, scriptText: string, config: Object): bo
 /**
  * @deprecated Use ruleTester
  */
-export function makeTest(rule: string, scripts: Array<string>, expected: boolean, config?: { rules: {} }) {
+export function makeTest(rule: string, scripts: Array<string>, expected: boolean, config?: Lint.Configuration.IConfigurationFile) {
```

Comment:
True, this is something I covered in the `ruleTester`. I will be leaving this one as is since I'm slowly changing the rest of the tests to use the ruleTester so that I may remove this file. I'll do one more update to one of the tests in which I spammed the boilerplate.